pkg/apps/apis/apps/v1/defaults_test.go

```diff
@@ -24,6 +26,7 @@ func TestDefaults(t *testing.T) {
 			original: &DeploymentConfig{},
 			expected: &DeploymentConfig{
 				Spec: DeploymentConfigSpec{
+					Replicas: &one,
```

Comment:
doesn't this mean the default changed from 0 to 1?

Reply:
Yeah, I suppose there may be people who rely on that.  Something we should have fixed during 3.6, but forgot.  Going to back this out.